crates/lunatic-control-axum/src/api.rs

```diff
@@ -42,6 +43,7 @@ impl ApiError {
             ApiError::InvalidData(_) => "invalid_data",
             ApiError::InvalidPathArg(_) => "invalid_path_arg",
             ApiError::InvalidQueryArg(_) => "invalid_query_arg",
+            ApiError::ProcessNotFound => "process_not_found",
```

Comment:
These are more HTTP errors, so we could simply add `NotFound(String)` and put info "Process not found" in it. It's already obvious from the context what HTTP 404 means. Maybe we don't even need the message, just `NotFound`

Reply:
I'm going to go with no-arg `NotFound` for the time being.